apps/docs/pages/guides/integrations/oauth-apps/creating-an-oauth-app.mdx

```diff
@@ -0,0 +1,25 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'creating-an-oauth-app',
+  title: 'Create an OAuth App (Beta)',
+  description: 'Create an OAuth App',
+}
+
+## Overview
+
+This guide steps through registering an OAuth app under your organization.
+
+## Creating an OAuth App
+
+1. In your organization's settings, navigate to the [**OAuth Apps**](https://app.supabase.com/org/_/apps) tab.
+2. In the upper-right section of the page, click **Add application**.
+3. In "Application name", fill in the name of your OAuth app.
+4. In your website URL, fill in the URL of the app that's using the OAuth app.
+5. Upload a logo for your OAuth app.
+6. In "Authorization callback URLs", add the callback URL of your app.
+7. At the bottom of the page, click **Confirm**.
```

Comment:
Yeah i think it would be great if we could add some screenshots to the UI but i've left those out as i think the UI might still be subject to change at this point